cranelift/isle/isle/src/ast.rs

```diff
@@ -419,3 +420,26 @@ pub enum ArgPolarity {
     /// *from* the extractor op.
     Output,
 }
+
+/// An implicit converter, which can appear in extractor or
+/// constructor position. This sets up an ambient automatic type
+/// conversion that is used when needed.
+#[derive(Clone, Debug, PartialEq, Eq)]
+pub struct Converter {
+    /// The Rust function name.
+    pub func: Ident,
```

Comment:
Is this assuming that the term used for converting is always an external Rust function? That seems like an unnecessary restriction to me.

Reply:
(commented below -- yes, I'll change this)